assets/components/cards/utils.ts

```diff
@@ -15,9 +15,80 @@ import LargePictureTextCard from 'components/cards/render/LargePictureTextCard';
 import EventsTwoByTwoCard from 'components/cards/render/EventsTwoByTwoCard';
 import NavigationSixPerRow from 'components/cards/render/NavigationSixPerRow';
 import {gettext} from 'utils';
+import {MoreNewsSearchKind} from './render/MoreNewsButton';
+import {ComponentType} from 'react';
 
+export interface IProps2x2 {
+    events: any;
+    title: string;
+}
 
-const CARD_TYPES = [
+export interface I6NavigationRow {
+    card: any;
+}
+
+export interface IProps4PhotoGallery {
+    photos: Array<any>;
+    title: string;
+    moreUrl: string;
+    moreUrlLabel: string;
+    listConfig: any;
+}
+
+export interface IPropsRest {
+    items: Array<any>;
+    title: string;
+    id: string;
+    openItem: any;
+    isActive: boolean;
+    cardId: string;
+    listConfig: string;
+    kind?: MoreNewsSearchKind;
+    type: string;
+}
+
+interface ICard1 {
+    _id: '2x2-events';
+    text: string;
+    editComponent: ComponentType<any>;
+    size: number;
+    dashboardComponent: ComponentType<IProps2x2>;
+}
+
+interface ICard2 {
```

Comment:
here it would be also better to use something more descriptive than `ICard1` `ICard2` etc